tably.py

```diff
@@ -120,15 +155,29 @@ def create_table(self, file):
                 units = get_units(self.units)
                 rows.insert(1, r'{0}{0}{1} \\'.format(indent, units))
 
-        header = HEADER.format(
+        content = '\n'.join(rows)
+        if not self.fragment:
+            header = HEADER.format(
             label=add_label(self.label, indent),
             caption=add_caption(self.caption, indent),
             align=format_alignment(self.align, len(columns)),
             indent=indent,
-        )
-        content = '\n'.join(rows)
-        footer = FOOTER.format(indent=indent)
-        return '\n'.join((header, content, footer))
+            )
+            footer = FOOTER.format(indent=indent)
+            return '\n'.join((header, content, footer))
+        else:
+            return content
+    def save_each_table(self, file, out):
```

Comment:
Nitpicking, but since a single table is passed, a better function name reflecting that would be `save_single_table` or something like that.